preamble-ts-dts/preamble-bdd-api.d.ts

```diff
@@ -0,0 +1,211 @@
+// Type definitions for Preamble-TS-Standalone
+// Project: https://github.com/Preamble-BDD/standalone/
+// Definitions by: Jeffrey Schwartz <https://github.com/jeffschwartz/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../q/Q.d.ts" />
+
+declare interface IMatcher {
```

Comment:
Are you question my use of prefacing the interface with "I" and suggesting that I shouldn't? Personally & IMHO it serves to distinguish between a class and an interface declaration. It is also a common practice in other languages as well. Eagerly awaiting your feedback on this :)